.ruby-version

```diff
@@ -0,0 +1 @@
+2.2.3
```

Comment:
why not 2.2.2 like described in the commit body?

Reply:
It isnt supported by Netlify https://www.netlify.com/docs/continuous-deployment/#ruby

Reply:
ah okay that was kind of irritating when reading the commit body^^